docusaurus/docs/protocol/primitives/claim_and_proof_lifecycle.md

```diff
@@ -126,6 +128,17 @@ gantt
 
 - [`poktroll.shared.Params` / `sharedtypes.Params`](https://github.com/pokt-network/poktroll/blob/main/proto/poktroll/shared/params.proto)
 
+### Claim Expiration
```

Comment:
General feedback (not actionable right now).

This sort of documentation is very accurate and very correct but it reminds me of IEEE RFCs: https://datatracker.ietf.org/doc/html/rfc7951

If we're doing is as a reference for ourselves and AIs, no changes.

As you iterate on docs in the future, just keep in mind that 99% of people will want to:

1. Read 1-2 sentences
2. See colors and titles
3. Look at pictures
4. Move on

Reply:
The deliverable was to capture this detail. This seemed like the most appropriate place to do so. I couldn't think of a way to make a picture out of this at the time.